src/components/organisms/EntityDetails/EntityDetailsContent/Settings/Settings.styled.tsx

```diff
@@ -1,17 +1,7 @@
-import {Form, Space} from 'antd';
-
 import {QuestionCircleOutlined} from '@ant-design/icons';
 
 import styled from 'styled-components';
 
-export const StyledSpace = styled(Space)`
-  width: 100%;
-`;
-
-export const StyledFormItem = styled(Form.Item)`
```

Comment:
Isn't the `margin-bottom: 0` needed? I believe that it's deleting the inconsistent empty space after the last (or the only = last) item in the configuration card.

Reply:
I used so called custom `<FormItem />` which already has `margin-bottom: 0` instead of this `<StyledFormItem />`. I checked all the pages and I did not affect anything.